packages/dockest/src/utils/createDefaultReadinessChecks.ts

```diff
@@ -0,0 +1,40 @@
+import { execaWrapper } from './execaWrapper'
+import { DefaultReadinessChecks, Runner } from '../@types'
+
+export const createDefaultReadinessChecks = ({
+  runner,
+  runner: {
+    containerId,
+    dockerComposeFileService: { ports },
+  },
+}: {
+  runner: Runner
+}): DefaultReadinessChecks => ({
+  postgres: async ({ database, username }: { database: string; username: string }) => {
```

Comment:
couldn't we default to `$POSTGRES_DB` and `$POSTGRES_USER`? Those should be available inside an official Postgres container: https://hub.docker.com/_/postgres